### PR TITLE
New Integration: Ubiquiti airCube

### PR DESCRIPTION
**Pull request by petroffc**
## Proposed change
Documentation for new Ubiquiti airCube device tracker integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31684
- This PR fixes or closes issue: N/A

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/aircube.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Ubiquiti airCube
+description: Instructions on how to integrate Ubiquiti airCube routers into Home Assistant.
+logo: ubiquiti.png
+ha_category:
+  - Presence Detection
+ha_release: 0.106.0
```
Comment (klaasnicolaas) [inline source/_integrations/aircube.markdown]:
```suggestion
ha_release: 0.106
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/aircube.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  required: true
+  type: string
+verify_ssl:
+  description: If SSL/TLS verification for HTTPS resources needs to be turned off (for self-signed certs, etc.)
+  required: false
+  type: boolean
+  default: true
+{% endconfiguration %}
+
+Note on SSL: The verify_ssl flag must currently be set to `False` as the certificate on the device cannot easily be updated. The device runs a modified version of OpenWrt with SSH disabled. While certificates can be managed in OpenWrt, Ubquiti, in violation of the GPL, has not released the source code for the airCube.
```
Comment (frenck) [inline source/_integrations/aircube.markdown]:
```suggestion
Note on SSL: The verify_ssl flag must currently be set to `False` as the certificate on the device cannot easily be updated. The device runs a modified version of OpenWrt with SSH disabled. While certificates can be managed in OpenWrt, Ubiquiti, in violation of the GPL, has not released the source code for the airCube.
```

Maybe convert this to an actual note?
<https://developers.home-assistant.io/docs/en/documentation_create_page.html#html>

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**